### PR TITLE
Finish User: show members of course and activities

### PR DESCRIPTION
**Pull request by trantube94**
I have created a pull request, check it please!

### CODE REVIEW

**Comment by hienbx94** [comment]
Ok

**Comment by ttuan** [comment]
OK

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
-    <li><%= link_to user_subject.subject.name, course_user_subject_path(@course, user_subject) %></li>
-  <% end %>
-</ul>
+<% unless @user_course.nil? %>
+  <div class="row">
+    <div class="col-md-9">
+      <subject class="col-md-12">
+        <h1><%= @course.name %></h1>
+        <%= t "course.line" %>
+        <% @user_subjects.each do |user_subject| %>
```
Comment (ThuBM) [inline app/views/courses/show.html.erb]: refactor đoạn code này

File: app/views/courses/show.html.erb [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
+        <h2><%= t "course.activities_in_course" %></h2> 
+        <%= render @activities %>
+        <%= will_paginate @activities %>
+      </activity>
+    </div>
+
+    <listuser class="col-md-3">
+      <h2><%= t "course.all_member" %></h2>
+      <% @user_courses.each do |user_course| %>
+        <% user = user_course.user %>
```
Comment (ThuBM) [inline app/views/courses/show.html.erb]: Không set data trong view

**Comment by ThuBM** [comment]
OK

**Pull request merged**